src/core/reducers/addons.js

```diff
@@ -174,6 +174,17 @@ export const createInternalPreviews = (
   }));
 };
 
+const selectLocalizedUrlWithOutgoing = (url, lang) => {
```

Comment:
Having some tests for this function would be nice. (sorry that I couldn't get back to this patch before).

Reply:
https://github.com/mozilla/addons-frontend/issues/9976